src/xcode/ENA/ENA/Source/Scenes/ENSetting/ExposureNotificationSettingViewController.swift

```diff
@@ -292,20 +278,45 @@ final class ExposureNotificationSettingViewController: UITableViewController, Ac
 		alert.addAction(UIAlertAction(title: AppStrings.ExposureNotificationSetting.privacyConsentDismissAction, style: .cancel, handler: { action in completionHandler(action) }))
 		self.present(alert, animated: true, completion: nil)
 	}
+	
+	private func bannerCell(for indexPath: IndexPath, in tableView: UITableView) -> UITableViewCell {
+		guard let cell = tableView.dequeueReusableCell(withIdentifier: ReusableCellIdentifier.banner.rawValue, for: indexPath) as? ImageTableViewCell else {
+			return UITableViewCell()
```

Comment:
Is displaying an empty cell expected behaviour? If not, I would prefer to fatalError here. Because not registering a cell, resulting in a crash here, is a programming error.